reana/reana_dev/git.py

```diff
@@ -82,6 +85,45 @@ def get_current_commit(srcdir):
     )
 
 
+def git_is_current_version_tagged(component):
+    """Determine whether the current version in source code is present as a git tag."""
+    tags_history = run_command(
+        "git tag --list", component, display=False, return_output=True
+    )
+    current_version = get_component_current_version_from_file(component)
+    return current_version in tags_history
+
+
+def git_create_release_commit(component):
+    """Create a release commit for the given component."""
+    if "release: v" in get_current_commit(get_srcdir(component)):
```

Comment:
It is good to check for `release: vXXX`, but when you run the `git-create-release-commit` function, the release branch being created (elsewhere in the code in line 121, not in this place) is called `release-0.7.0a4` without leading `v`. Shall we use leading `v` in branch names for consistency?

Reply:
I still get `release-0.7.0a4` branch names, so this does not seem to be resolved. Shall we use or remove leading `v` everywhere except git tags? Let's sync IRL.